bangank36/zoomify-generator
# 🗺️ Zoomify Generator: provide Openseadragon

## Summary
This PR is simple, it display the tiles on openseadragon viewer

## Reference
#15 

## Testing
- Check against localhost:8000/openseadragon.html
- Confirm all tiles display correctly
